frontend/app/model/Eventbrite.scala

```diff
@@ -207,10 +207,17 @@ object Eventbrite {
     val limitedAvailabilityText = "Last tickets remaining"
     val isLimitedAvailability = internalTicketing.exists(event => event.ticketsNotSold <= Config.eventbriteLimitedAvailabilityCutoff && !event.isSoldOut)
     val ticketsNotSold = internalTicketing.map(_.ticketsNotSold)
-
     val isSoldOut = internalTicketing.exists(_.isSoldOut)
-    val isBookable = status == "live" && !isSoldOut
-    val isPastEvent = status != "live" && status != "draft"
+
+    val isBookable = {
+      val hasBookableTicketClasses = ticket_classes.exists(_.sales_end < DateTime.now)
```

Comment:
Could we rename this variable to `isStartedAndHasBookableTicketClasses`, making it equal `status == "started" && ticket_classes.exists(_.sales_end < DateTime.now)`? The fact that it's only true when `status == "started"` lowers it's possibly disruptive impact, and the ugly name makes it clear people shouldn't mess with it.

As for comparing with `sales_start`, you can if you like, but I think it's probably not essential given we check against  `status == "started"`.